src/App.tsx

```diff
@@ -41,7 +42,7 @@ const App: React.FC = () => {
   }, [])
 
   useEagerConnect()
-  useFetchPublicData()
+  // useFetchPublicData()
```

Comment:
Needed for testnet to function. Once contract is deployed on main net I will remove